poc/TestOfTestFrameworkByReference/NFUnitTestByReference.nfproj

```diff
@@ -49,6 +50,9 @@
     <Reference Include="nanoFramework.Runtime.Native">
       <HintPath>..\packages\nanoFramework.Runtime.Native.1.6.12\lib\nanoFramework.Runtime.Native.dll</HintPath>
     </Reference>
+    <Reference Include="nanoFramework.System.Runtime">
```

Comment:
This project is referencing the TestFramework, therefore doesn't need to have a reference to the Runtime nuget.